tap_github/streams.py

```diff
@@ -0,0 +1,102 @@
+from enum import Enum
+from typing import Type, Set, List
+
+from singer_sdk.streams.core import Stream
+
+from tap_github.organization_streams import OrganizationStream
+from tap_github.repository_streams import (
+    AnonymousContributorsStream,
+    CommitsStream,
+    CommunityProfileStream,
+    ContributorsStream,
+    EventsStream,
+    IssueCommentsStream,
+    IssueEventsStream,
+    IssuesStream,
+    LanguagesStream,
+    PullRequestsStream,
+    ReadmeHtmlStream,
+    ReadmeStream,
+    RepositoryStream,
+    StargazersStream,
+    StatsContributorsStream,
+    AssigneesStream,
+    CollaboratorsStream,
+    ReviewsStream,
+    ReviewCommentsStream,
+    ProjectsStream,
+    ProjectColumnsStream,
+    ProjectCardsStream,
+    PullRequestCommits,
+    MilestonesStream,
+    CommitCommentsStream,
+    ReleasesStream,
+)
+from tap_github.user_streams import (
+    StarredStream,
+    UserContributedToStream,
+    UserStream,
+)
+from tap_github.organization_streams import (
+    OrganizationStream,
+    TeamsStream,
+    TeamMembersStream,
+    TeamRolesStream,
+)
+
+
+class Streams(Enum):
```

Comment:
I'm confused by why this derives from `Enum`.

Reply:
Without `Enum` I wouldn't get access to features like defining methods to the enum values. E.g: `Streams.ORGANIZATIONS.all_valid_queries()`